themes/powerline/powerline.theme.bash

```diff
@@ -2,10 +2,10 @@
 
 . "$BASH_IT/themes/powerline/powerline.base.bash"
 
-PROMPT_CHAR=${POWERLINE_PROMPT_CHAR:=""}
```

Comment:
These are not broken, they display fine for me in the Atom editor - these are unicode characters, see the screenshot below:

![image](https://user-images.githubusercontent.com/907063/33820103-f3c91d2e-de4d-11e7-8d95-7c21a2f059d6.png)

Reply:
Yep, I can do that. Thanks I didn't realize I was missing fonts. 

I'll see if I can replace the screenshots from the documentation too. The [screenshot](https://camo.githubusercontent.com/e142ebc7b54816e8335ba66991ec8fe9cbceb6af/68747470733a2f2f7261772e6769746875622e636f6d2f77696b692f726576616e732f626173682d69742f696d616765732f73637265656e73686f74732f706f7765726c696e652d626c61636b2e6a7067) on the Themes wiki shows broken characters. I thought it confirmed by suspicion that characters were broken. 

Thanks again.